cmd/gomodfs/gomodfs-main.go

```diff
@@ -35,6 +35,8 @@ var (
 	flagMemLimitMB = flag.Int64("mem-limit-mb", 0, "how many megabytes (MiB) of memory gomodfs can use to store file contents in memory; 0 means to use a default")
 	portmapper     = flag.Bool("portmapper", false, "if set, run rpcbind portmapper on TCP+UDP port 111 (needed for Windows NFS clients). For NFS mode only")
 	flagWinFSP     = flag.Bool("winfsp", false, "if set, use WinFSP on Windows")
+
+	flagNFSForWindows = flag.Bool("nfs-for-windows-clients", runtime.GOOS == "windows", "if set, optimize NFS server behavior for Windows clients")
```

Comment:
IIRC are we hoping to just use this as a temporary measure? Maybe worth a brief comment if so

Reply:
I'll add a comment in any case, but basically: ideally we wouldn't have this, but since we don't plan to use this in production anyway, it didn't seem worth fixing and making it explicit instead. This basically only exists for CI tests now, just to keep it from bitrotting.